lib/order.rb

```diff
@@ -33,7 +33,7 @@ def confirm_order
 
   def time_plus_1hour
     t = Time.new
-    hour_plus1 = ((t.strftime("%H").to_i) + 1)
+    hour_plus1 = (t.strftime("%H").to_i + 1)
     t.strftime("at #{hour_plus1}:%M%p")
   end
 
```

Comment:
Incrementing time by one hour seems a bit overcomplicated, could be done in shorter way.